lib/testers/rule-tester.js

```diff
@@ -429,7 +429,7 @@ RuleTester.prototype = {
                     item.errors.length, item.errors.length === 1 ? "" : "s", messages.length, util.inspect(messages)));
 
                 for (let i = 0, l = item.errors.length; i < l; i++) {
-                    assert.ok(!("fatal" in messages[i]), `A fatal parsing error occurred: ${messages[i].message}`);
+                    assert.ok(!messages[i].fatal, `A fatal parsing error occurred: ${messages[i].message}`);
```

Comment:
The last minor added the complete source code of a file to the output, so I don't think that's the issue, or at least it's not thought of as an issue now.

Should the json formatter be more than just `JSON.stringify`? Filter out e.g. source code